extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -125,60 +122,78 @@ export class AmpAdXOriginIframeHandler {
       this.sendEmbedInfo_(this.baseInstance_.isInViewport());
     }));
 
+    // Iframe.onload normally called by the Ad after full load.
+    const iframeLoadPromise = loadPromise(this.iframe).then(() => {
+      // Wait just a little to allow messages to arrive.
```

Comment:
i see. I guess we're only interested no-content at this point. can you just update the doc:
```
 // Wait just a little to allow no-content to arrive.
```